@kiva/kv-shop/src/basketCredits.ts

```diff
@@ -74,7 +74,7 @@ export async function applyPromoCredit(
 		variables: { ...options?.variables },
 	});
 
-	return !!data?.shop?.addCreditByType;
+	return data?.shop;
```

Comment:
This is going to need to return `data` at the root b/c the `errors` object will live alongside the `shop` node...
ex.
```
{"data":{"shop":{"id":"GQiC3v8YCSq73smPLsQLXQ==","addCreditByType":null,"__typename":"ShopMutation"}},"errors":[{"message":"The promotion for (dogs) does not exist, but you can still use your own credit to checkout.","locations":[{"line":1,"column":146}],"path":["shop","addCreditByType"],"extensions":{"code":"upc.promoDoesNotExist"}}]}
```

Reply:
done

Reply:
Oh shoot... we actually have to return the full response b/c `data` and `errors` are siblings :(

Reply:
I got that response by using a fake upc code to force failure in our existing test page. you can inspect the network panel to find that call and it's result.